src/guild.cpp

```diff
@@ -81,3 +81,12 @@ void Guild::addRank(uint32_t rankId, const std::string& rankName, uint8_t level)
 {
 	ranks.emplace_back(std::make_shared<GuildRank>(rankId, rankName, level));
 }
+
+void Guild::setBankBalance(uint64_t balance) {
+	bankBalance = balance;
+	Database& db = Database::getInstance();
+	std::ostringstream query;
+	query << "UPDATE `guilds` SET `balance`='" << bankBalance << "' ";
+	query << "WHERE `id`='" << id << "'";
```

Comment:
Why adding quotes around integer values?

Reply:
MySQL schema standard convention (etc take a look at INSERT INTO statements in schema.sql), but I see its different in the source, and strictly speaking uneccesary, so will fix.